iwyu.cc

```diff
@@ -2673,6 +2674,22 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     return true;
   }
 
+  bool VisitArrayType(ArrayType* type) {
+    if (CanIgnoreCurrentASTNode())
+      return true;
+    if (!compiler()->getLangOpts().CPlusPlus)
+      ReportTypeUse(CurrentLoc(), &*type->getElementType(), DerefKind::None);
+    return true;
+  }
+
+  bool VisitIncompleteArrayType(IncompleteArrayType *type) {
```

Comment:
What is it needed for? `IncompleteArrayType` is derived from `ArrayType`, so the previous handler should be sufficient.

Reply:
nvm turns out this isn't necessary after all